app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -25,6 +29,14 @@ const validatePreference = validate({
   city: validate.isPresent("is required"),
   state: validate.isPresent("is required"),
   zipCode: validate.isPresent("is required")
+},
+{ // If this applies, run the above validation on that field.
```

Comment:
I see what's going on here, but it seems pretty confusing for me. i'm going to see if I can think of a clearer way to do it.